usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -11,7 +11,10 @@ is_true "$GRUB_RESCUE" || return
 # (grub-probe or grub2-probe only exist in GRUB 2)
 # in particular do not run this script when GRUB Legacy is used
 # (for GRUB Legacy output/default/94_grub_rescue.sh is run):
-type -p grub-probe >&2 || type -p grub2-probe >&2 || { LogPrint "Skipping GRUB_RESCUE setup for GRUB 2 (no GRUB 2 found)." ; return ; }
+if [[ ! $( type -p grub-probe ) && ! $( type -p grub2-probe ) ]]; then
```

Comment:
No need to correct it.
I merged it as is.

It is a matter of personal taste what should be in the log.

In my old log it was:
<pre>
++ type -p grub-probe
++ type -p grub2-probe
/usr/sbin/grub2-probe
</pre>

In the current log it is:
<pre>
+++ type -p grub-probe
++ [[ ! -n '' ]]
+++ type -p grub2-probe
++ [[ ! -n /usr/sbin/grub2-probe ]]
</pre>

For my personaly eyes the curren log looks
less clean that the old one - but again - this
is a matter of personal taste.